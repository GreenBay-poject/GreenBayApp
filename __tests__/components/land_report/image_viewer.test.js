import React from 'react';
import renderer from 'react-test-renderer';
import Image_Viewer from '../../../components/land_report/image_viewer';

describe('<Image_Viewer />', () => {
    it('Sample Image_Viewer', () => {
        const tree = renderer.create(<Image_Viewer latitude={67.3} longitude={34.65} date={'2021-11-02'} image={"https://apktada.com/storage/images/com/johartest/com.johartest_1.png"} setimage={null} setstep={null} />).toJSON();
        expect(tree).toMatchSnapshot();
        expect(tree.type).toBe('View');
        expect(tree.children[0].type).toBe("View");
        expect(tree.children[1].type).toBe("View");
        expect(tree.children[2].type).toBe("Image");
        console.log(tree)

    });
});