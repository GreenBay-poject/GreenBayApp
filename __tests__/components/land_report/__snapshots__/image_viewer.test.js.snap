// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Image_Viewer /> Sample Image_Viewer 1`] = `
<View>
  <View
    style={
      Object {
        "marginBottom": 0,
        "marginTop": 0,
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "alignSelf": "center",
          "backgroundColor": "#153d37",
          "borderRadius": 10,
          "opacity": 1,
          "padding": 10,
          "width": "95%",
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#FFFFFF",
            "fontSize": 14,
          }
        }
      >
        GET REPORT
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "marginBottom": 2,
        "marginTop": 2,
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "alignSelf": "center",
          "backgroundColor": "#153d37",
          "borderRadius": 10,
          "opacity": 1,
          "padding": 10,
          "width": "95%",
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#FFFFFF",
            "fontSize": 14,
          }
        }
      >
        BACK
      </Text>
    </View>
  </View>
  <Image
    source={
      Object {
        "uri": "https://apktada.com/storage/images/com/johartest/com.johartest_1.png",
      }
    }
    style={
      Object {
        "aspectRatio": 1,
        "height": undefined,
        "width": "100%",
      }
    }
  />
</View>
`;
